raterapi/models/game.py

```diff
@@ -9,3 +10,15 @@ class Game(models.Model):
     time_to_play = models.IntegerField()
     age = models.IntegerField()
     categories = models.ManyToManyField("Categories", related_name="game_categories", related_query_name="game_category")
+
+    @property
+    def avg_rating(self):
+        ratings = Ratings.objects.filter(game__id=self.id)
+
+        if len(ratings):
+            sum_of_ratings = 0
+            for rating in ratings:
+                sum_of_ratings += rating.value
```

Comment:
`rating.value` should be `rating.rating` so it matches what's in your Ratings model